tests/hurl/feetransfers.hurl

```diff
@@ -29,3 +29,20 @@ jsonpath "$.data.feetransfers[0].blockHeight" == 120
 jsonpath "$.data.feetransfers[0].canonical" == true
 jsonpath "$.data.feetransfers[0].dateTime" == "2021-03-17T08:39:00.000Z"
 duration < 1000
+
+POST {{url}}
+```graphql
+{
+  feetransfers(sortBy: BLOCKHEIGHT_DESC, query: {recipient: "B62qjWwDxk5nGMXN32ffuMpMSp3wPa7BLB3AXHH5EYiczcRxDTF9uP4", canonical: true}) {
+    fee
+    recipient
+    type
+    dateTime
+  }
+}
+```
+HTTP 200
+[Asserts]
+jsonpath "$.data.feetransfers" count == 1
```

Comment:
Will there only ever be 1 or do we need to also query by epoch?

Reply:
This will only every be 1 in the first 120 blocks. Also, filtering by epoch doesn't make sense in this cause as it's not a staking ledger. I could narrow it down by including a staking ledger if that's what you want